### PR TITLE
Remove reference to 'user cohorts'

### PR DESCRIPTION
**Pull request by StevenC365**
The removed section referred to user cohorts, which I assume was some proposed feature to group together users and set their co-existence mode. Remove this section as there is no such thing as a user cohort.

### CODE REVIEW

**Comment by LolaJacobsen** [comment]
@LaszloSomi could you review these changes please?

**Comment by LolaJacobsen** [comment]
@bjwhalen could you review these changes please?

**Comment by bjwhalen** [comment]
why are we taking out the staged approach?

**Comment by StevenC365** [comment]
Hi BJ,

That section talks about ‘adding your users to a cohort’ and ‘selecting a cohort and assigning an upgrade policy’. It’s as if at some point there was a plan to build a mechanism called cohorts for progressive upgrades. We’ve had users asking us in the Tech Community where the option for cohorts is.

That section of the article was quite specific, it wasn’t just talking about a concept of grouping your users together, my guess this was a planned feature that never made it to production?

Steven

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by LaszloSomi** [comment]
I would just remove the cohort part and not the whole section. You do it for one user or a group of users it is the same step.

**Comment by StevenC365** [comment]
Every step refers to either creating a cohort or selecting a cohort through the admin portal. No such concept or interface exists. It's not reasonable to assume a user would manually update the coexistence state of every user in their tenant one by one. There could be a script at this point to apply policy to the members of a group. What's there today is incorrect and misleading, and users in the TechCommunity have noticed.

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Teams/upgrade-to-Teams-execute-SkypeforBusinessOnline.md [hunk 1]
```diff
@@ -56,38 +56,8 @@ Follow these steps to upgrade all of your users to Teams at one time.
...
-2. Next to **Teams upgrade**, select **Edit**.
-3. In the **Teams Upgrade** pane, under **Coexistence mode**, select **Teams Only** from the drop-down list.
-
 ## Phone System and Teams upgrade
 
 If your Skype for Business Online deployment includes Phone System with Calling Plans and Microsoft is your public switched telephone network (PSTN) provider, upgrading your users to Teams will automatically transition inbound PSTN calling to Teams.
 
-If your Skype for Business Online deployment includes Phone System with Cloud Connector Edition, see the [additional considerations for Phone System Direct Routing](2-envision-make-my-service-decisions-direct-routing.md).
\ No newline at end of file
+If your Skype for Business Online deployment includes Phone System with Cloud Connector Edition, see the [additional considerations for Phone System Direct Routing](2-envision-make-my-service-decisions-direct-routing.md).
```
Comment (nenonix) [inline Teams/upgrade-to-Teams-execute-SkypeforBusinessOnline.md]:
```suggestion
If your Skype for Business Online deployment includes Phone System with Cloud Connector Edition, see [additional considerations for Phone System Direct Routing](2-envision-make-my-service-decisions-direct-routing.md).
```

Comment (LaszloSomi) [reply Teams/upgrade-to-Teams-execute-SkypeforBusinessOnline.md]: Is that the only change? Works for me.

**Comment by scanum** [comment]
@LolaJacobsen @bjwhalen @LaszloSomi @StevenC365 Hi! any updates on this PR? Changes were made, is this PR ok to be merged? If yes, please merge. Thanks!!

**Comment by LolaJacobsen** [comment]
Waiting for @bjwhalen input.

**Comment by scanum** [comment]
@bjwhalen @SerdarSoysal  Any updates on this?

File: Teams/upgrade-to-Teams-execute-SkypeforBusinessOnline.md (hunk 1 above)
Comment (bjwhalen) [reply Teams/upgrade-to-Teams-execute-SkypeforBusinessOnline.md]: we must not cut the section "upgrade users in stages".  That is critical and frankly a better way to go than all at once.    If there are concerns with the word "cohort" then simply change that word or phrase. How about "waves" of users, or something like that.

**Comment by LolaJacobsen** [comment]
Per @bjwhalen, it is NOT OK to remove the user cohort section. I'm closing this without merging.

**Comment by bjwhalen** [comment]
but, there are some incremental things that do need to be changed. I agree the notion of cohorts doesn't exist in UX. I'll do a separate PR to address that.

**Pull request closed without merging**